draft-ietf-quic-transport.md

```diff
@@ -1553,16 +1553,20 @@ token with a newer one. The client MUST NOT use the token provided in a Retry
 for future connections. Servers MAY discard any Initial packet that does not
 carry the expected token.
 
-Unlike the token that is created for a Retry packet, there might be some time
-between when the token is created and when the token is subsequently used.
-Thus, a token SHOULD include an expiration time.  The server MAY include either
-an explicit expiration time or an issued timestamp and dynamically calculate the
-expiration time.  It is also unlikely that the client port number is the same on
-two different connections; validating the port is therefore unlikely to be
-successful.
-
 A token SHOULD be constructed for the server to easily distinguish it from
 tokens that are sent in Retry packets as they are carried in the same field.
+The token SHOULD NOT expose linkability; i.e., information that lets observers
```

Comment:
This is worth making a new paragraph for.  How about:

"The token MUST NOT include information that would allow it to be linked to the connection on which it was issued by an on-path observer.  For example, it cannot include the connection ID or addressing information unless the values are encrypted."

Reply:
+1, but slight grammar: "The token MUST NOT include information that would allow it to be linked by an on-path observer to the connection on which it was issued."

Reply:
Applied in 67df7d1.